libbeat/processors/add_host_metadata/add_host_metadata.go

```diff
@@ -71,10 +84,68 @@ func (p *addHostMetadata) loadData() {
 		if p.info.OS.Build != "" {
 			p.data.Put("host.os.build", p.info.OS.Build)
 		}
+
+		if p.config.NetInfoEnabled {
+			// IP-address and MAC-address
+			var ipList, hwList, err = p.getNetInfo()
+			if err != nil {
+				logp.Warn("Error when getting network information %v", err)
```

Comment:
We prefer not to use `Warn` as it's not clear if this is something that needs action. I suggest we set it to `Info` for now.

Reply:
Ok!